website/blog/2023-06-14-install-ferretdb-mongodb-alternative-ubuntu.mdx

```diff
@@ -0,0 +1,321 @@
+---
+slug: install-ferretdb-mongodb-alternative-ubuntu
+title: 'How to Install FerretDB on Ubuntu'
+author: Klinsmann Öteyo
+description: >
+  This is a guide on how to install FerretDB - the MongoDB alternative - on Ubuntu.
+image: /img/blog/install-ferretdb-ubuntu.jpg
+keywords: [FerretDB, mongodb alternative]
+---
+
+<head>
+  <link rel="canonical" href="https://tutornix.com/how-to-install-ferretdb-mongodb-alternative-on-ubuntu-22-0420-04/" />
+</head>
+
+![How to install FerretDB on Ubuntu](/img/blog/install-ferretdb-ubuntu.jpg)
+
+_This is a duplicate post from [Tutornix](https://tutornix.com/how-to-install-ferretdb-mongodb-alternative-on-ubuntu-22-0420-04/) about installing FerretDB on Ubuntu; we are grateful to the author for the permission to repost this on our blog._
```

Comment:
The word "duplicate" sounds a bit strange to me. That's typically called reblogging / reposting. We could also say "copy".